### PR TITLE
cnap: add event log replay and verify RTMR value with measurement

### PR DESCRIPTION
**Pull request by Hulongyin**
This PR is to add event log replay and verify RTMR value with measurement.

### CODE REVIEW

File: cnap/core/keybroker.py [hunk 1]
```diff
@@ -68,13 +70,14 @@ class SimpleKeyBrokerClient(KeyBrokerClientBase):
           key (SWK) to encrypt the user key (wrapped_key).
         - Encrypt the SWK by the public key from client (wrapped_swk).
     For a key broker client, here is an example flow to get a key from KBS:
+        - Verify Event Logs.
```
Comment (dongx1x) [inline cnap/core/keybroker.py]: => Get and replay all event logs, and verify by the measurement register.

File: cnap/core/eventlog.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        LOG.info("Fetch measurements in IMR[%d]", index)
+        imr_measurement = base64.b64decode(Measurement.get_platform_measurement(
+            MeasurementType.TYPE_TDX_RTMR, None, index))
+        LOG.info("IMR[%d](measurement): %s", index, imr_measurement.hex())
+
+        # Get IMR value from replayed event log
+        if index not in measurement_dict or measurement_dict[index] == {}:
+            LOG.error("IMR[%d] verify failed, the replayed value from event logs doesn't exist",
+                index)
+            flag = False
```
Comment (dongx1x) [inline cnap/core/eventlog.py]: return False

File: cnap/core/eventlog.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        for value in measurement_dict[index].values():
+            imr_replayed = value
+            break
+
+        LOG.info("IMR[%d](replayed): %s", index, imr_replayed.hex())
+        if imr_measurement == imr_replayed:
+            LOG.info("IMR[%d] passed the verification.", index)
+        else:
+            LOG.error("IMR[%d] did not pass the verification.", index)
+            flag = False
```
Comment (dongx1x) [inline cnap/core/eventlog.py]: return False

File: cnap/core/keybroker.py (hunk 1 above)
Comment (Hulongyin) [reply cnap/core/keybroker.py]: Thanks, fixed.

File: cnap/core/eventlog.py (hunk 2 above)
Comment (Hulongyin) [reply cnap/core/eventlog.py]: Thanks, fixed.

File: cnap/core/eventlog.py (hunk 3 above)
Comment (Hulongyin) [reply cnap/core/eventlog.py]: Thanks, fixed.

**Review by dongx1x — APPROVED** [review]
LGTM.

**Pull request merged**